packages/regl-worldview/src/commands/Command.js

```diff
@@ -81,6 +82,37 @@ export default class Command<T> extends React.Component<Props<T>> {
     }
   }
 
+  getDrawPropFromHitmapId(id: number) {
+    const { hitmapProps } = this.props;
+    for (const drawProp of hitmapProps) {
+      if (drawProp.color) {
+        const drawPropId = getIdFromColor(drawProp.color.map((color) => color * 255));
+        if (drawPropId === id) {
+          return drawProp;
+        }
+      }
+    }
+    return null;
+  }
+
+  fireRelevantMouseHandler(id: number, e: any, ray: any, mouseEventName: MouseEventEnum) {
+    const mouseHandler = this.props[mouseEventName];
+    if (!mouseHandler) {
+      return;
+    }
+
+    const drawProp = this.getDrawPropFromHitmapId(id);
```

Comment:
The naming is a little confusing, technically we are returning the `hitmapProp`, not the `drawProp`? What's your consideration  on the type of prop to return?  If we are going to handle hitmapId internally, would it make sense to just return the `drawProp`?

Reply:
good point, lemme rename it to `hitmapProp`.